changelogs/unreleased/gh-10460-on_replication_split_brain_rollback-event.md

```diff
@@ -0,0 +1,6 @@
+## feature/box
+
+* Added a new `box.ctl.on_replication_split_brain_rollback` event that occurs
+  when asynchronously committed transaction stored in the synchronous
+  transaction queue get rolled back by a PROMOTE request to prevent split brain
```

Comment:
```suggestion
* Added a new `box.ctl.on_replication_split_brain_rollback` event that occurs
  when an asynchronously committed transaction stored in the synchronous
  transaction queue gets rolled back by a PROMOTE request to prevent split brain
```